manifests/install.pp

```diff
@@ -14,9 +14,9 @@
   $install_path        = $::kibana::install_path,
 ) {
 
-  $filename         = $::architecture ? {
-    'i386'  => "kibana-${version}-linux-x86",
-    'amd64' => "kibana-${version}-linux-x64",
+  $filename = $::architecture ? {
+    'i386'           => "kibana-${version}-linux-x86",
```

Comment:
Can you add x86 for 32bit? Because:

> Gentoo also calls “i386” “x86”

See: http://docs.puppetlabs.com/facter/2.4/core_facts.html#architecture.

Sorry for introducing this issue.